README.md

```diff
@@ -1,2 +1,2 @@
 # dragonslave
-Lina lover's only
+Lina lovers only
```

Comment:
Great change! The possessive case of "lover" did not work here because "only" is an adjective.